database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -346,13 +416,20 @@ private void executeInitQueries(Connection connection, List<String> initQueries)
    */
   public abstract static class DBSinkConfig extends DBConfig {
     public static final String TABLE_NAME = "tableName";
+    public static final String DB_SCHEMA_NAME = "dbSchemaName";
     public static final String TRANSACTION_ISOLATION_LEVEL = "transactionIsolationLevel";
 
     @Name(TABLE_NAME)
     @Description("Name of the database table to write to.")
     @Macro
     public String tableName;
 
+    @Name(DB_SCHEMA_NAME)
+    @Description("Name of the database table to write to.")
```

Comment:
nit: Name of the database schema to write to.

Reply:
This property will be visible to all the databases even if they do not support the notion of Schema. As far as I know SQL server and oracle has notion of schema, but MySQL does not. Should we add it to Oracle for now and then incrementally add to other databases such as SQL server, Postgres etc?

Reply:
Correct. I figure that all sinks will always default to the users schema. In many customers I've worked with in the past there may be one ETL user that has write access across many schemas/databases. The way everything is written, it will always default to just that users default DB. I figure we can expose over Oracle and then if there's requests we can expose to other sinks.

Reply:
oh you mean since we only change the widget json for oracle sink this wont be visible to other sinks? It might show up for other sinks too but not just in neat format. better to validate that.

Reply:
correct. The only place I'm using that specific widget is in the oracle sink. I spot checked some others and I didn't see it creep in there...at least with the 1.6 build